doc/api/intl.md

```diff
@@ -0,0 +1,198 @@
+| [`String.prototype.normalize()`][]      | none (function is no-op)          | full                         | full                   | full
+| `String.prototype.to*Case()`            | full                              | full                         | full                   | full
+| [`Intl`][]                              | none (object does not exist)      | partial/full (depends on OS) | partial (English-only) | full
+| [`String.prototype.localeCompare()`][]  | partial (not locale-aware)        | full                         | full                   | full
+| `String.prototype.toLocale*Case()`      | partial (not locale-aware)        | full                         | full                   | full
+| [`Number.prototype.toLocaleString()`][] | partial (not locale-aware)        | partial/full (depends on OS) | partial (English-only) | full
+| `Date.prototype.toLocale*String()`      | partial (not locale-aware)        | partial/full (depends on OS) | partial (English-only) | full
+
+*Note*: The "(not locale-aware)" designation denotes that the function carries
+out its operation just like the non-`Locale` version of the function, if one
+exists. For example, under `none` mode, `Date.prototype.toLocaleString()`'s
+operation is identical to that of `Date.prototype.toString()`.
+
+### Disable all internationalization features (`none`)
+
+If this option is chosen, most internationalization features mentioned above
+will be **unavailable** in the resulting `node` binary.
+
+### Build with a pre-installed ICU (`system-icu`)
+
+Node.js can link against an ICU build already installed on the system. In fact,
+most Linux distributions already come with ICU installed, and this option would
+make it possible to reuse the same set of data used by other components in the
+OS.
+
+Functionalities that only require the ICU library itself, such as
+[`String.prototype.normalize()`][], are fully supported under `system-icu`.
+Features that require ICU locale data in addition, such as
+[`Intl.DateTimeFormat`][] *may* be fully or partially supported, depending on
+the completeness of the ICU data installed on the system.
+
+### Embed a limited set of ICU data (`small-icu`)
+
+This option makes the resulting binary link against the ICU library statically,
+and includes a subset of ICU data (typically only the English locale) within
+the `node` executable.
+
+Functionalities that only require the ICU library itself, such as
+[`String.prototype.normalize()`][], are fully supported under `small-icu`.
+Features that require ICU locale data in addition, such as
+[`Intl.DateTimeFormat`][], generally only work with the English locale:
+
+```js
+const january = new Date(9e8);
+const english = new Intl.DateTimeFormat('en', { month: 'long' });
+const spanish = new Intl.DateTimeFormat('es', { month: 'long' });
+
+console.log(english.format(january));
+// Prints "January"
+console.log(spanish.format(january));
+// Prints "January" on small-icu
```

Comment:
Default v6.11.3 prints `M01` for me.

Reply:
Hmm, it's M01 for me on master as well but January on v6.x. Can you check the output after changing your `LC_ALL` environment variable to `en_US`?

Reply:
<details>
<summary>Output:</summary>

```js
> set LC_ALL=en_US

> set LC_ALL
LC_ALL=en_US

> node.6.11.3.v8-5.1.exe
> const january = new Date(9e8);
> const english = new Intl.DateTimeFormat('en', { month: 'long' });
> const spanish = new Intl.DateTimeFormat('es', { month: 'long' });
> console.log(english.format(january));
January
> // Prints "January"
> console.log(spanish.format(january));
M01
> // Prints "January" on small-icu
> // Should print "enero"
```
</details><br>

Reply:
FWIW, I use the standalone binary from https://nodejs.org/download/release/latest-v6.x/win-x64/

Reply:
Ah you are on Windows. That's different...

Reply:
So maybe we can state `"January" or "M01"`?

Reply:
@vsemozhetbyt Sounds like a good idea.